integration/serviceWorker/serviceWorker.js

```diff
@@ -14,7 +14,7 @@
 * limitations under the License.
 */
 
-importScripts('../../../dist/package/firebase-app.js');
+importScripts('./node_modules/firebase/firebase-app.js');
```

Comment:
I `npm install` the firebase package that we build in `dist` here so we can consume it as a third party would for the test.